odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -7,8 +7,9 @@
         <t t-if="not object.name and object.parent_name">
             <span itemprop="name" t-esc="object.parent_name"/>
         </t>
-        <t t-if="options.get('country_image') and 'country_id' in fields and object.country_id and object.country_id.image">
-            <span t-field="object.country_id.image" t-options='{"widget": "image", "class": "country_flag"}'/>
+        <t t-if="options.get('country_image') and 'country_id' in fields and object.country_id and object.country_id.image_url">
+            <img t-attf-src="#{object.country_id.image_url}"
+                class="o_country_flag" width="50%"/>
```

Comment:
Not exactly sure where this is used but the 50% width seems a bit random.

Also, class was changed from "country_flag" to "o_country_flag", but I guess that's intended?